src/server/game/Globals/ConversationDataStore.cpp

```diff
@@ -103,18 +105,37 @@ void ConversationDataStore::LoadConversationTemplates()
 
             uint32 conversationId = fields[0].GetUInt32();
             uint32 actorId = fields[1].GetUInt32();
-            uint16 idx = fields[2].GetUInt16();
+            uint64 actorGuid = fields[2].GetUInt64();
+            uint16 idx = fields[3].GetUInt16();
 
             if (ConversationActorTemplate const* conversationActorTemplate = Trinity::Containers::MapGetValuePtr(_conversationActorTemplateStore, actorId))
             {
+                if (actorGuid != 0)
+                    TC_LOG_ERROR("sql.sql", "Table `conversation_actors` references both actor (ID: %u) and actorGuid (GUID: " UI64FMTD ") for Conversation %u, GUID is ignored", actorId, actorGuid, conversationId);
```

Comment:
move this message above the if condition, and check what your message says

Reply:
It shall just print this error.
Due to other if clause the guid is ignored, See some lines below.